NETPROGRAM/BasicExercise/Exercise05.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NETPROGRAM.BasicExercise
+{
+    public class Exercise05
+    {
+   
+        public bool CheckNumberRequest(int n)
+        {
+            int length = n.ToString().Length;
+            int result = 0;
+            int number = n;
+
+            do
+            {
+
+                result += (int) Math.Pow(number % 10, length);
```

Comment:
Hàm build-in này dùng với double, số nguyên thì nên tự implement lấy
```suggestion
                result += (int) Math.Pow(number % 10, length);
```

Reply:
done